resilience4j-spring-boot2/build.gradle

```diff
@@ -1,6 +1,9 @@
 dependencies {
     compile ( libraries.spring_boot2_aop )
     compile ( libraries.spring_boot2_actuator )
+    compileOnly ( libraries.spring_boot2_config_processor )
+    compile project(':resilience4j-annotations')
+    compile project(':resilience4j-spring')
```

Comment:
I hope the dependency to `resilience4j-spring` does not clash with Spring Boot2.
`resilience4j-spring` uses Spring Framework 4 to compile.
Spring Boot uses Spring Framework 5.

Reply:
You can either run `./gradlew resilience4j-spring-boot-2:dependencies` or
`./gradlew resilience4j-spring-boot-2:dependencyInsight --configuration compile`

Reply:
I run ./gradlew resilience4j-spring-boot2:dependencies | grep -B 1 springframework | grep 4.3 and got no result